doc/source/user_guide/io.rst

```diff
@@ -3310,10 +3310,10 @@ applications (CTRL-V on many operating systems). Here we illustrate writing a
 
 .. code-block:: python
 
-    >>> df = pd.DataFrame({'A': [1, 2, 3],
-    ...                    'B': [4, 5, 6],
-    ...                    'C': ['p', 'q', 'r']},
-    ...                   index=['x', 'y', 'z'])
+    >>> df = pd.DataFrame({"A": [1, 2, 3],
+    ...                    "B": [4, 5, 6],
+    ...                    "C": ["p", "q", "r"]},
+    ...                   index=["x", "y", "z"])
```

Comment:
Can you use black-style line wrapping here?

Reply:
Added an additional commit below to address this, how does that look?

Reply:
Looks good to me, thanks